packages/types/src/user.ts

```diff
@@ -39,9 +40,15 @@ export interface UserResource extends ClerkResource {
   isPrimaryIdentification: (ident: EmailAddressResource | PhoneNumberResource) => boolean;
   getSessions: () => Promise<SessionWithActivitiesResource[]>;
   setProfileImage: (params: SetProfileImageParams) => Promise<ImageResource>;
-  createExternalAccount: ({ strategy, redirect_url }: { strategy: OAuthStrategy, redirect_url?: string }) => Promise<ExternalAccountResource>;
-  get verifiedExternalAccounts(): ExternalAccountResource[]
-  get unverifiedExternalAccounts(): ExternalAccountResource[]
+  createExternalAccount: ({
+    strategy,
```

Comment:
Not related to this PR but just came to my attention, we should probably update the types here to something like `CreateExternalAccountParams`, in order to match the rest of the codebase :)